src/app/screens/CreatePassword.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { FormEvent } from 'react';
+
+import { createPassword, useDispatch } from '@store';
+import { passwordStrength } from 'check-password-strength';
+import { object, refine, string } from 'superstruct';
+import { FormError, useForm } from 'typed-react-form';
+
+import { getValidator } from '@app/utils';
+import warning from '@assets/icons/circle-warning.svg';
+import {
+  Body,
+  Box,
+  Button,
+  Flex,
+  FormInput,
+  Heading,
+  Image,
+  Label,
+  PanelBottom
+} from '@components';
+import { getKBHelpArticle, KB_HELP_ARTICLE } from '@config/helpArticles';
+import { translate, translateRaw } from '@translations';
+import { PasswordStrength } from '@types';
+
+const CREATE_PASSWORD_STRUCT = object({
+  password: refine(string(), 'Strong password', (value) => {
+    if (passwordStrength(value).id === PasswordStrength.STRONG) {
+      return true;
+    }
+
+    return translateRaw('PASSWORD_TOO_WEAK');
+  }),
+  passwordConfirmation: refine(string(), 'Equal to password', (value, context) => {
+    return value === context.branch[0].password;
```

Comment:
This needs a better explainer text.

`Expected a value of type string, but received:"password1"`